WireUI/Sources/WireSettingsUI/Account/Backup/Views/BackupActionsView.swift

```diff
@@ -0,0 +1,100 @@
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireDesign
+import WireReusableUIComponents
+
+public struct BackupActionsView: View {
+    @ObservedObject private var viewModel: BackupActionsViewModel
+    @State private var isBackupSheetPresented: Bool = false
+    @State private var isRestoreSheetPresented: Bool = false
+
+    public init(viewModel: BackupActionsViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        List {
+            ForEach(viewModel.sections) { section in
+                Section(
+                    footer: section.type.footer
+                ) {
+                    Button(action: {
+                        switch section.type {
+                        case .backup:
+                            isBackupSheetPresented.toggle()
+                        case .restore:
+                            isRestoreSheetPresented.toggle()
+                        }
+                    }, label: {
+                        HStack {
+                            section.type.title
+                                .font(.textStyle(.body2))
+                                .foregroundStyle(Color.primaryText)
+                            Spacer()
+                            Image(systemName: "chevron.right").foregroundStyle(Color.primary)
+                        }
+                    })
+                    .sheet(isPresented: $isBackupSheetPresented) {
+                        NavigationStack {
+                            ExportBackupView(
+                                passwordValidator: viewModel.passwordValidator,
+                                exportBackup: { password in
+                                    viewModel.backupActiveAccount(password: password)
+                                }
+                            )
+                        }
+                        .presentationDetents([.medium, .large])
+                    }
+                }
+            }
+        }
+        .listStyle(.grouped)
+        .listRowBackground(Color(ColorTheme.Backgrounds.background))
+    }
+}
+
+#Preview {
+    BackupActionsView(viewModel: BackupActionsViewModel(
+        backupSource: MockBackupSource(),
+        backupHandler: BackupHandler(
+            onSuccess: { _, _  in },
+            onFailure: { _ in }
+        ),
+        passwordValidator: MockBackupPasswordValidator()
+    ))
+}
+
+private class MockBackupSource: BackupSourceProtocol {
+    func backupActiveAccount(password: String) throws -> URL {
+        URL(fileURLWithPath: "path")
+    }
+
+    func clearPreviousBackups() {}
+}
+
+class MockBackupPasswordValidator: BackupPasswordValidatorProtocol {
+    func isValid(password: String) -> Bool {
+        true
+    }
+
+    var localizedRulesDescription: String {
+        "Use at least 8 characters, with one lowercase letter, one capital letter, a number, and a special character."
+    }
+}
```

Comment:
suggestion: 
create a dedicated file for preview source code, something like `BackupActionsView.preview.swift` and just call it from here:

```suggestion
#Preview {
    BackupActionsViewPreview()
}
```

Reply:
implemented